app/src/pages/inside/filtersPage/filterPageToolbar/filterPageToolbar.jsx

```diff
@@ -30,10 +30,8 @@ const messages = defineMessages({
   validate: ({ filter }) => ({
     filter: filter && filter.length < 3 ? 'filterNameError' : undefined,
   }),
-  onChange: (values, dispatch, props, previousValues) => {
-    if (typeof previousValues.filter === 'undefined') {
-      return;
-    }
+  enableReinitialize: true,
+  onChange: (values, dispatch, props) => {
     if (!values.filter || values.filter.length >= 3) {
```

Comment:
Let's use common validator for this and [that row](https://github.com/reportportal/service-ui/pull/1964/files#diff-e984a29f971b56d9cb7558ab94889c44L31)

Reply:
Fixed